Packs/QRadar/Integrations/QRadar_v3/QRadar_v3_test.py

```diff
@@ -128,6 +128,7 @@ def test_connection_errors_recovers(mocker):
      - Ensure that success message is printed and recovery for http request happens.
     """
     mocker.patch.object(demisto, "error")
+    mocker.patch("QRadar_v3.time.sleep")
```

Comment:
why do we need sleep statement in the code?

Reply:
its to mock the sleep, for handling connection errors

Reply:
otherwise it just takes longer for the test to execute which is a waste of time